src/common/modules/data/DefaultSettings.js

```diff
@@ -11,12 +11,22 @@
  * @const
  * @type {Object}
  */
+
+// checks for OS dark theme and returns the appropriate background color
+ const setQrBackgroundColor = () => {
```

Comment:
As per [our guide](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#js):
> Avoid anonymous functions, which have no name (i.e. not really assigned) unless they do really do simple things.

(I've edited the md file now to clarify that a little…)

Generally, don't use arrow functions here. Either inline it into the actual object (I would choose so) or make it a real "usual" function and JSDOC-document it.

Reply:
Actually _in this case_ you may even be able to simplify that whole thing to _one_ line: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
Sorry, read that in the guide but forgot. Setting to ternary (but pulling `window.matchMedia('(prefers-color-scheme: dark)').matches` out into a `const` so the line doesn't go on forever)